controller/modules/equipment/api.go

```diff
@@ -119,7 +119,8 @@ func (e *Controller) LoadAPI(r *mux.Router) {
 
 //swagger:model equipmentAction
 type EquipmentAction struct {
-	On bool `json:"on"`
+	On            bool `json:"on"`
+	StayOffOnBoot bool `json:"stay_off_on_boot"`
```

Comment:
Not sure why this is required. This attribute is already present on equipment, why we need it again in EquipmentAction

Reply:
Removed property StayOffOnBoot as requested by R. Dey.